examples/inference/api_server_openai/query_openai_sdk.py

```diff
@@ -41,21 +41,47 @@
 
 args = parser.parse_args()
 
-client = OpenAI()
-# # List all models.
-models = client.models.list()
-print(models.data, "\n")
-
-# Note: not all arguments are currently supported and will be ignored by the backend.
-chat_completion = client.chat.completions.create(
-    model=args.model_name,
-    messages=[
-        {"role": "assistant", "content": "You are a helpful assistant."},
-        {"role": "user", "content": "Tell me a long story with many words."},
-    ],
-    stream=args.streaming_response,
-    max_tokens=args.max_new_tokens,
-    temperature=args.temperature,
-    top_p=args.top_p,
-)
-print(chat_completion)
+client = OpenAI(base_url="http://localhost:8000/v1", api_key="not_needed")
+
+
+def stream_chat():
+    for chunk in client.chat.completions.create(
+        model=args.model_name,
+        messages=[{"role": "user", "content": "What is better sorting algorithm than quick sort?"}],
+        stream=True,
+        max_tokens=args.max_new_tokens,
+        temperature=args.temperature,
+        top_p=args.top_p,
+    ):
+        content = chunk.choices[0].delta.content
```

Comment:
Is it necessary to verify content of the chunk here? such as chunk != None and chunk !=""